<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MVPropertyWidget</class>
 <widget class="QWidget" name="MVPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>名称</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>音量调节</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="volumeSpinBox"/>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="volumeSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>列表</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="maximumSize">
        <size>
         <width>37</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_remove">
       <property name="maximumSize">
        <size>
         <width>37</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>移除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clean">
       <property name="maximumSize">
        <size>
         <width>37</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_upper">
       <property name="maximumSize">
        <size>
         <width>37</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>上移</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_lower">
       <property name="maximumSize">
        <size>
         <width>37</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>下移</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>148</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
